### ISSUE TITLE
Stop ranging / monitoring?

### ISSUE THREAD

**Issue by AndrewPetrovics** [loup-v/beacons#3]
Does cancelling a subscription automatically stop the ranging and monitoring on the device (i.e. call stopMonitoringForRegion / stopRangingForRegion)?

Or is there another way to do this?

Thanks!

**Comment by lukaspili** [comment]
Yes, when the latest subscription is cancelled, it automatically stops the ongoing request (ranging or monitoring).

**Comment by AndrewPetrovics** [comment]
Awesome! Thank you sir.
